integration-tests/tracecontext/docker/Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN unzip trace-context.zip
 RUN rm trace-context.zip
 RUN mv trace-context-${TRACECONTEXT_GIT_TAG}/test /tracecontext-testsuite
 
-FROM python:3.13.5-slim@sha256:f2fdaec50160418e0c2867ba3e254755edd067171725886d5d303fd7057bbf81
+FROM python:3.13.5-slim@sha256:6544e0e002b40ae0f59bc3618b07c1e48064c4faed3a15ae2fbd2e8f663e8283
```

Comment:
weird that a sha would change for the same version, but here it is https://hub.docker.com/layers/library/python/3.13/images/sha256-09aa001329f10fc6d784a1a48bba50815bc35bee2f31b6268a5557a9d78ae764?context=explore